allensdk/brain_observatory/behavior/session_apis/abcs/__init__.py

```diff
@@ -1,2 +1,4 @@
 from allensdk.brain_observatory.behavior.session_apis.abcs.behavior_base import BehaviorBase  # noqa: F401, E501
 from allensdk.brain_observatory.behavior.session_apis.abcs.behavior_ophys_base import BehaviorOphysBase  # noqa: F401, E501
+from allensdk.brain_observatory.behavior.session_apis.abcs.behavior_raw_data_base import BehaviorRawDataBase  # noqa: F401, E501
+from allensdk.brain_observatory.behavior.session_apis.abcs.behavior_ophys_raw_data_base import BehaviorOphysRawDataBase  # noqa: F401, E5
```

Comment:
Should that last line also have E501 at the end, not E5?

Reply:
Good catch, I've updated!